recipes/asciimatics/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "asciimatics" %}
+{% set version = "1.7.1" %}
```

Comment:
Looks like the latest version is 1.9.0. Any reason not to start with that?

Reply:
I decided to start with version [1.7.1](https://pypi.org/project/asciimatics/1.7.1/#files) because it is the last version that was upload as a tar file to PyPI. Version [1.9.0](https://pypi.org/project/asciimatics/1.9.0/#files) was uploaded as a wheel file.  I made this decision after reading the documentation on [Writing the meta.yaml](https://conda-forge.org/docs/meta.html) stating that Conda-forge requires building from tarballs. If there is a way to build from the wheel file, I'm happy to adjust the recipe accordingly.